bot/src/message_handler.rs

```diff
@@ -128,25 +127,28 @@ async fn get_sqlx_database_client() -> Result<sqlx::PgPool> {
     Ok(pool)
 }
 
-fn get_update_language_code(update: &Update) -> String {
-    let mut language_code = "en".to_string();
+fn get_update_language_code(update: &Update) -> Language {
+    let mut language_code = Language::En;
 
     if let UpdateKind::Message(message) = &update.kind {
         if let Some(user) = &message.from() {
             if let Some(language_code_option) = &user.language_code {
-                language_code = language_code_option.to_string();
+                language_code = Language::from_str(language_code_option).unwrap_or(Language::En);
```

Comment:
Let's keep it as it is. But later we may add a warning to the log.

Reply:
I didn't get that point